pkg/controllers/disruption/singlenodeconsolidation.go

```diff
@@ -51,7 +51,8 @@ func (s *SingleNodeConsolidation) ComputeCommand(ctx context.Context, disruption
 	// Set a timeout
 	timeout := s.clock.Now().Add(SingleNodeConsolidationTimeoutDuration)
 	constrainedByBudgets := false
-
+	//Set ConsolidationTimeoutsTotal to zero
+	ConsolidationTimeoutsTotal.Add(0, map[string]string{consolidationTypeLabel: s.ConsolidationType()})
```

Comment:
Can we set this in the `init()` where we create the metrics rather than in the actual call-path? I think we can just do a `ConsolidationTimeousTotal.Set(0)` IIRC

Reply:
So ConsolidationTimeousTotal is a [counter metric](https://pkg.go.dev/github.com/awslabs/operatorpkg@v0.0.0-20241130193813-da0c28954330/metrics#CounterMetric) so it does not have set(). Also could you direct me to the init() you are referring to here? I cannot seem to find it.

Reply:
`Add(0)` seems fine then